.dockerignore

```diff
@@ -1,3 +1,5 @@
 pybin
 __pycache__
 config.txt
+lesson_data/
+.venv/
```

Comment:
based on the 🚫 symbol github is showing us, please check your editor to make sure that you always have newlines at the end of the file, and also update this PR to includes those newlines. (apologies, I realize one or more of the files was missing one previously)